CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+Hello. This is the place where you can contribute ideas for an update to the Ansible docsite.
+
+We love it when people submit new ideas, make improvements, and fix our mistakes. So welcome!
+
+## Things to know before your first contribution
+
+- All code and doc submissions are done through pull requests against the `main` branch.
+- Take care to make sure no merge commits are in your submission. Use `git rebase` vs `git merge` for this reason.
```

Comment:
This only makes sense if the native merge button is used for PRs. If only squash is allowed, demanding this is pointless since the result would be the same, for the project, but the contributors would have to deal with more burden.
So far, I haven't seen anybody using merge on PRs in this repo.
If anybody's going to use differing merge strategies, this could just state that the maintainers will always use squash-"merge" if a PR has merge commits but if it's linear, preserving the history is up to the maintainers' discretion.

Reply:
Good point. I think this line was inherited from somewhere else. I'll drop it.